tests/lib/prepare.sh

```diff
@@ -69,7 +79,11 @@ EOF
 }
 
 prepare_classic() {
-    apt_install_local ${GOPATH}/snapd_*.deb
+    if [ "$SRU_VALIDATION" = "1" ]; then
```

Comment:
This smells like we want a helper like `install_build_snapd` or something here to avoid the copy/paste in multiple places. WDYT?